packages/utils/src/chromium-finder.ts

```diff
@@ -0,0 +1,388 @@
+            Browser.Chromium, [
+                `${path.sep}Chromium${path.sep}Application${path.sep}chromium.exe`
+            ]
+        ],
+        [
+            // TODO: Not sure how to identify the browser correctly, I believe it depends on which one you install first?
+            Browser.Edge, [
+                // Canary suffix path
+                `${path.sep}Microsoft${path.sep}Edge SxS${path.sep}Application${path.sep}msedge.exe`,
+                `${path.sep}Microsoft${path.sep}Edge${path.sep}Application${path.sep}msedge.exe`,
+                // Dev suffix path
+                `${path.sep}Microsoft${path.sep}Edge Dev SxS${path.sep}Application${path.sep}msedge.exe`,
+                `${path.sep}Microsoft${path.sep}Edge Dev${path.sep}Application${path.sep}msedge.exe`
+            ]
+        ]
+    ])]
+]);
+
+
+/** Find chromium executables on macOS */
+const darwin = (browser: Browser) => {
+    const platformBroserInfo = browserVariables.get('darwin');
+    const suffixes = platformBroserInfo!.get(browser);
+
+    if (!suffixes) {
+        throw new Error(ERRORS.NotSupportedBrowser);
+    }
+
+    /**
+     * The following generates a list of all the installed applications in macOS.
+     * The `grep` command makes sure we only get the lines that end with `.app`
+     * because `-dump` generates more information than the one needed.
+     * The `awk` commands makes sure we remove the initial part of the line so we
+     * only get the path.
+     *
+     * More info in:
+     * http://krypted.com/mac-security/lsregister-associating-file-types-in-mac-os-x/
+     */
+    const LSREGISTER = '/System/Library/Frameworks/CoreServices.framework' +
+        '/Versions/A/Frameworks/LaunchServices.framework' +
+        '/Versions/A/Support/lsregister';
+
+    const lines = execSync(
+        `${LSREGISTER} -dump` +
+        ` | grep -i '\.app$'` + // eslint-disable-line
+        ` | awk '{$1=""; print $0}'`)
+        .toString()
+        .split(newLineRegex);
+
+    /**
+     * Because `suffixes` are already prioritized, we iterate them first to make
+     * sure that priorization is respected
+     */
+    for (const suffix of suffixes) {
+        for (const inst of lines) {
+
+            const execPath = path.join(inst.trim(), suffix);
+
+            /**
+             * An example of valid path for Chrome would be:
+             * `/Applications/Google Chrome.app/Contents/MacOS/Google Chrome`
+             */
+            if (canAccess(execPath)) {
+                return execPath;
+            }
+        }
+    }
+
+    return '';
+};
+
+/** Find chromium executables on Linux. */
+const findChromeExecutable = (folder: string, regExes: string[]) => {
+    const argumentsRegex = /(^[^ ]+).*/; // Take everything up to the first space
+
+    for (const browserRegex of regExes) {
+
+        const chromeExecRegex = `^Exec=/.*/${browserRegex}-.*`;
+
+        if (canAccess(folder)) {
```

Comment:
if you have restored `isFile`, then is not going to work as expected, right?

Reply:
It's going to work as expected. I'm just using the same name of the variable as `chrome-finder` upstream. In their code they assume everything they check for is a file so this is fine.

Reply:
so `folder` is a file?

Reply:
It should be a file, don't know why they call it folder but now I'm hesitating. This is for Linux so not easy for me to test 🤔

Reply:
So this should probably use `is-dir`. Nice catch.